### DISCUSSION TITLE
v2ray 可以进行多节点自动选择吗？

### DISCUSSION THREAD

**Discussion by aszswaz** [v2fly/v2ray-core#1646] [Q&A]
v2ray 可以进行多节点自动选择吗？

假设我有一个节点在美国洛杉矶，一个节点在日本东京，我优先使用洛杉矶节点，v2ray 可以在洛杉矶节点宕机的情况下，自动连接日本东京的节点吗？

**Comment by ping2me** [comment]
小火箭应该有这个功能

**Reply by aszswaz** [reply]
是有这功能，但 v2ray 好像是把流量均分到所有的节点上，对我的影响就是，我有一个阿里云日本节点丢包率很高，但我在美国洛杉矶的节点良好。有很多流量被分摊到日本的节点去，造成整体的网页加载速度很慢。
